### ISSUE TITLE
Cannot find filed with multiple extentions

### ISSUE THREAD

**Issue by kadikraman** [facebook/flow#4378]
We have some files in the codebase with filenames like `fileName.decorator.js`. These get a "required module not found" error with flow.
Have tried editing the options as
```
[options]
module.file_ext=.decorator
```
and also
```
[options]
module.file_ext=.decorator.js
```
but neither seems to be doing the trick. Was wondering if I'm missing something?

Thanks a lot!

**Comment by TrySound** [comment]
Why not just js?

**Comment by kadikraman** [comment]
Fair point. Not a fan of it, but it's an existing convention in the codebase so not up to me to change at this point.

**Comment by TrySound** [comment]
I mean you do not have types in other javascript files?

**Comment by kadikraman** [comment]
We do. Those resolve fine. It's just the `.[something].js` ones that don't resolve with flow. E.g:

<img width="741" alt="screen shot 2017-07-13 at 17 22 39" src="https://user-images.githubusercontent.com/6534400/28176595-f5a3141c-67ef-11e7-9b37-8a4ba8d76294.png">

Found this while digging through the source code: https://github.com/facebook/flow/blob/master/tests/config_file_extensions/.flowconfig

Seems like it only works if the last value isn't `.js`? I've managed to get filenames ending just with `.sql` working fine using the `module.file_ext` option.

**Comment by GezzMC** [comment]
I have a similar problem. My workaround is easy. I just move my `bignumber.js` folder.

I get the error `Cannot resolve module bignumber.js` when I use file `node_modules/bignumber.js.js` to point to folder `node_modules/misc_1/bignumber.js`, like this:

    module.exports = require("./misc_1/bignumber.js")

I do something similar with about 20 other JS files, among which is `underscore.string.js`. The problem is with the `.js.js` extension.

Thanks.
